src/main/webapp/test/mentors.test.js

```diff
@@ -0,0 +1,67 @@
+import {getMentors, createMentorElement, createEmailLink, createLocationHeader, createImg, createBioParagraph, createNameHeader, createGitHubLink, createInterestTagsDiv} from '../mentors.js';
+import '@testing-library/jest-dom/extend-expect';
+
+global.mailtouiApp = {run: () => {}};
+
+const mockMentor = {
+  interestTags: ['interest1', 'interest2'],
+  projects: [{respositoryID: 'mockRepoID', name: 'mockProjectName'}],
+  bio: 'This is the mock bio.',
+  email: 'samialves@google.com',
+  gitHubID: 'mockGitID',
+  image: '../images/dior.jpg',
+  location: 'Mockville, Mocktana',
+  name: 'Mocky the Mockster',
+};
+
+describe('Mentor card', () => {
+  it('is created', () => {
+    const mentorElement = createMentorElement(mockMentor);
+    expect(mentorElement).not.toBeNull();
+  });
+
+  it('has the correct image element', () => {
+    const imgElement = createImg(mockMentor.image);
+    expect(imgElement.getAttribute('class')).toBe('mentor-picture');
```

Comment:
Do you think it is worth it to test the classes as well, I feel like they change pretty often. As a bit of a middle ground, maybe making the classes into fixtures. like 
```JavaScript
const IMAGE_CLASS = "blah-blah-image";
...
expect(imageElement.getAttribute('class')).toBe(IMAGE_CLASS);
```
Honestly, that seems like a lot of work and refactoring. This is good as it is in my opinion. I feel like it might be a bit difficult to maintain in the long run though.

Reply:
I am really interested in your suggestion of fixtures and in Richie's suggestion of only testing for the presence of certain classes. I will look into both options.